index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>My Portolio Project</title>
+</head>
+<body>
+
+  
+        <header class ="page-header">
+            <div>
+              <a id="logo-text" href="welcome">Welcome</a>
+            </div>  
+            <nav id="navbar">
+            <i class="fa fa-bars"></i>
+            </nav>
+        </header>
+
+    <main class="main-class">    
+        <section class="introduction-text">
+            <div>
+               <h1 id="Greeting-text">Hey there. I’m Shedrack!</h1>
+               <p id="profession-text"> I’m a software Engineer.</p>
+            </div>
+
+
+        </section>
+        <div class="description">
+            <p>
+                I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see, which i assure you is high quality and you have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+        </div>
+    
+        <div class="icon-bar">
+            <a href="#">
+              <i class="fa fa-twitter"></i>
+            </a>   
+                    
+            <a href="#">
+              <i class="fa fa-linkedin"></i>
+            </a>
+                    
+                
+            <a href="#">
+              <i class="fa fa-medium"></i>
+            </a>
+                        
+            
+            <a href="#">
+              <i class="fa fa-github"></i>
+            </a> 
+                        
+            <a href="#">
+              <i class="fa fa-angellist"></i>
+            </a>
+    
+        </div>
```

Comment:
Great UI @Shedrack-Sunday 🤝

- [ ] As suggested by the previous reviewers, kindly wrap this section in a `ul` instead of a `div`, and put `li` tags inside. This is necessary for semantic reasons. Kindly implement this change as requested. If you add styling properly, you can still come up with the intended design

Reply:
I've mentioned this severally. It will change the design. And I've used my last review for this project. Can you use empathy? Because none of the use seems to even take note of my messages. Not even a reply.

Reply:
Hello @Shedrack-Sunday.

- [ ] We follow review guidelines. I saw your issues from the previous reviewers. We are mandated to review based on what previous reviewers suggest. I cannot overlook what a previous reviewer has suggested. The requirement requested is implementable and your design can still be achieved with styling.

Reply:
His comments were" As you listed all social media icons using a ul element instead of div, please make sure each icon is wrapped in a li element"

Meaning there was no need for li is I used div instead of ul. It's fine. Just check the details well